app/models/like.rb

```diff
@@ -1,2 +1,8 @@
 class Like < ApplicationRecord
+  belongs_to :user, foreign_key: 'user_id'
+  belongs_to :post, foreign_key: 'post_id'
+
+  def update_comments_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ Optional ] I think it would be nice if you make use of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_likes_counter` method every time a new `Like` has been saved to the database, then make the `update_likes_counter` method private. Please see the example below for a suggestion on how to achieve this.

## Example code
```ruby
  class Like < ApplicationRecord
  belongs_to :user, foreign_key: 'user_id'
  belongs_to :post, foreign_key: 'post_id'

  after_save :update_likes_counter

  private

  def update_likes_counter
    post.update(likes_counter: post.likes.count)
  end
end

```